### ISSUE TITLE
JSX Syntax

### ISSUE THREAD

**Issue by mattjennings** [edbentley/replay#25]
Have you considered supporting JSX syntax? I was thinking of making a babel plugin, but I'm curious if you've either gone down that road already or decided against it.

**Comment by mattjennings** [comment]
I gave a quick go at implementing JSX using the `@babel/plugin-transform-react-jsx` plugin and a bit of webpack configuration. Ideally this would all be done in a babel, but for now you can check it out at https://github.com/mattjennings/replay-with-jsx

**Comment by edbentley** [comment]
Thanks for bringing this up, the plugin looks awesome!

My initial plan was to support JSX and I did some prototyping around it. However I found the greatest benefit JSX brings is more readable nesting - and since Replay doesn’t make use of the nesting, the extra build step (e.g. your Babel plugin) didn’t make it worth using.

I think if you prefer JSX it’s great you have options like your plugin, but I kind of prefer the existing way to just use functions and avoid the build step.
